### DISCUSSION TITLE
Client swell store graphql timeout!!

### DISCUSSION THREAD

**Discussion by tarikrital** [swellstores/community#518] [Help] [answered]
Few days ago Horizon theme build was working as normal but Yesterday and Today it timeout! what is the reason!

![image](https://github.com/swellstores/community/assets/52907282/7d22c944-1983-433f-945e-d56da308fb2b)

**Comment by logeshswell** [comment]
Are you still getting the timeout error?

**Reply by logeshswell** [reply]
I can see that the timeouts occur rarely twice a day. Let me take a deeper look at this issue and get back to you if I get some updates.

**Reply by tarikrital** [reply]
I noticed you selected the answer many times! (in this thread and other by me) let's keep the thread more obvious for other clients. (I think the answer should be selected by who created the discussion!)

Regarding the timeout problem, Today it is working ok and run faster than before.

I contrast, let us know the reason of that issue to avoid it in the future!

**Reply by logeshswell** [reply]
> Regarding the timeout problem, Today it is working ok and run faster than before.

Yes, our infrastructure team identified an issue with the service restart and applied a fix for it. Hence, you should experience comparatively better and improved API performance now.

**Reply by logeshswell** [reply]
> I think the answer should be selected by who created the discussion!

The answer can be selected by anyone who finds it right. I marked it as an **Answered** to help new people find a solution for this issue easier.

**Reply by logeshswell** [reply]
If the issue is resolved completely, the discussion will be marked as **Closed** after the confirmation from the people who raised the discussion.

**Reply by logeshswell** [reply]
Also, please feel free to reopen the discussion (or) unmark an answer anytime if you find it incorrect.

**Reply by tarikrital** [reply]
> Yes, our infrastructure team identified an issue with the service restart and applied a fix for it. Hence, you should experience comparatively better and improved API performance now.

This happened Today too!

**Reply by logeshswell** [reply]
@wpplumber We are actively working on it.

**Reply by tarikrital** [reply]
✅

**Reply by logeshswell** [reply]
We applied a fix for it and continuously monitoring it.
Please check and confirm if it's resolved for you.
Thanks!

**Reply by tarikrital** [reply]
Yes, resolved.

Thank you.

**Reply by tarikrital** [reply]
Again the issue back!

**Reply by logeshswell** [reply, marked as answer]
Yes, we had an infra issue that could have caused the timeout.
The issue was identified and fixed in a couple of hours.
For more info: https://status.swell.store/incidents/dz9ktfgcvzfr
